tests/test-cases/Group1-Docker-Commands/1-43-Docker-CP-Offline.robot

```diff
@@ -57,6 +57,28 @@ Clean up test files and VIC appliance to test server
     Cleanup VIC Appliance On Test Server
 
 *** Test Cases ***
+Try To Exploit VCH With Offline Copy of Malicious Tarball
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} pull ${busybox}
+    Should Be Equal As Integers  ${rc}  0
+
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} create --name exploitme ${busybox}
+    Should Be Equal As Integers  ${rc}  0
+
+    ${rc}  ${output}=  Run And Return Rc And Output  ls ${CURDIR}/*.tar.gz
+    Should Be Equal As Integers  ${rc}  0
```

Comment:
don't really need this it is presumably handled by the No such file check below

Reply:
oh, yeah, good catch this is leftover from testing my test (making sure I had put the archive in the right place and that the tests could see it etc)